common/app/routes/challenges/Test-Suite.jsx

```diff
@@ -39,6 +47,24 @@ export default class TestSuite extends PureComponent {
     });
   }
 
+  getAccessibleText(err, pass, text) {
```

Comment:
@BerkeleyTrue I'm sorry. I'm not really that experienced with JSX, so I didn't think there'd be any problem with it. I've updated the pull request per your comments -making the function top-level within the file, popping it out of the class, and verifying that npm test runs nicely and the results are still the same.

Thank you!